docs/index.bs

```diff
@@ -207,7 +208,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
     A [=/service worker=] has an associated <dfn>all fetch listeners are empty flag</dfn>. It is initially unset.
 
-    A [=/service worker=] has an associated <dfn>list of router rules</dfn>. It is initially an empty [=list=].
+    A [=/service worker=] has an associated <dfn>list of router rules</dfn> (a [=list=]) whose [=list/item=] is a {{RouterRule}}. It is initially an empty [=list=].
```

Comment:
```suggestion
    A [=/service worker=] has an associated <dfn>list of router rules</dfn> (a [=list=] of {{RouterRule}}s). It is initially an empty [=list=].
```

seems to match https://pr-preview.s3.amazonaws.com/sisidovski/ServiceWorkerForStaticRoutingAPI/pull/10.html#dfn-job-list-of-equivalent-jobs